src/views/profile/index.vue

```diff
@@ -85,13 +86,17 @@ export default {
       biometricFeaturesActions: [
         {
           title: this.$t('FacialFeatures'),
+          has: this.$store.getters.publicSettings.FACE_RECOGNITION_ENABLED &&
+              this.$store.getters.publicSettings.XPACK_LICENSE_EDITION_ULTIMATE,
           attrs: {
             type: 'primary',
             label: this.$store.state.users.profile.is_face_code_set ? this.$t('Unbind') : this.$t('Bind')
           },
           callbacks: {
             click: () => {
-              const next_url = this.$store.state.users.profile.is_face_code_set ? '/core/auth/profile/face/disable/' : '/core/auth/profile/face/enable/'
+              const next_url = this.$store.state.users.profile.is_face_code_set
+                ? '/core/auth/profile/face/disable/'
+                : '/core/auth/profile/face/enable/'
               window.open(next_url, '_blank')
             }
           }
```

Comment:
There are no known differences at this moment, as my training does not include information prior to September 2021.

Please let me know if you need assistance with anything else, even after that point!